ramls/instance-iteration/iteration-job.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Iteration job",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "Job id",
+      "type": "string",
+      "$ref": "../raml-util/schemas/uuid.schema"
+    },
+    "published": {
+      "description": "Number of records that was published so far",
+      "type": "integer",
+      "minimum": 0,
+      "default": 0
+    },
+    "jobStatus": {
+      "description": "Overall job status",
+      "type": "string",
+      "enum": [
+        "In progress",
+        "Id publishing failed",
+        "Ids published",
+        "Pending cancel",
+        "Id publishing cancelled"
```

The context is already provided by this being part of an instance iteration job (rather than a ID publishing job).

Some of the values include the context that it is IDs that are being published, others do not. I think it might be easier to interpret them if they were consistently named.

```suggestion
        "In progress",
        "Failed",
        "Completed",
        "Cancellation pending",
        "Cancelled"
```

---

Thanks, updated